test/unit/full_text_search/attachment_test.rb

```diff
@@ -104,6 +104,8 @@ def teardown_server
     def fixture_file_path(name)
       dir = File.dirname(__FILE__)
       path = Pathname(File.join(dir, "..", "..", "files", name)).expand_path
+      return path if Rails::VERSION::MAJOR >= 7
+
       fixture_path = Pathname(self.class.fixture_path)
       fixture_path += "files" if Rails::VERSION::MAJOR >= 6
       path.relative_path_from(fixture_path)
```

Comment:
Can we always use absolute path?

```suggestion
      Pathname(File.join(dir, "..", "..", "files", name)).expand_path
```

Reply:
Yes, we can do. I fixed it using the absolute path in any situation.

- Rails v5.2(Redmine v4.2): [ActionDispatch::TestProcess::FixtureFile#fixture_file_upload](https://github.com/rails/rails/blob/5-2-stable/actionpack/lib/action_dispatch/testing/test_process.rb)
- Rails v6.1(Redmine v5.0 and v5.1): [ActionDispatch::TestProcess::FixtureFile#fixture_file_upload](https://github.com/rails/rails/blob/6-1-stable/actionpack/lib/action_dispatch/testing/test_process.rb)
- Rails v7.1(Redmine master): [ActionDispatch::TestProcess::FixtureFile#fixture_file_upload](https://github.com/rails/rails/blob/7-1-stable/actionpack/lib/action_dispatch/testing/test_process.rb)